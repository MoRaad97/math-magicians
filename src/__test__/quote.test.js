import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../Components/quote';

it('render the Calculator Component', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
