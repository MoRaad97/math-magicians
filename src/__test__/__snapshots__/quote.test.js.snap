// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the Calculator Component 1`] = `
<section
  className="quote"
>
  <div
    className="container"
  >
    <p
      style={
        Object {
          "fontSize": "25px",
        }
      }
    >
      Mathematics are the result of mysterious powers which no one understands, and which the unconscious recognition of beauty must play an important part. Out of an infinity of designs a mathematician chooses one pattern for beauty's sake and pulls it down to earth.
      <br />
      <span>
         â€” Marston Morse, American mathematician 
      </span>
    </p>
  </div>
</section>
`;
